client/js/models/archive.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   namespace: 'archive',
   state: {
     key: null,
-    file: null
+    file: null,
+    numPeers: 0,
+    signalhubs: [
+      'signalhub.mafintosh.com',
+      'signalhub.dat.land'
```

Comment:
I guess a clearer way to say that is signalhubs should not be hard-coded on client, this should probably just be an empty array that gets populated from rehydration object.